cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/rolling/RackRolling.java

```diff
@@ -388,26 +194,24 @@ enum NodeFlavour {
                     .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
             return nextController(reconciliation, nodesNeedingRestart, activeControllerId, totalNumOfControllerNodes, quorumState);
 
-        } else if (partitioned.get(NodeFlavour.BROKER_AND_NOT_ACTIVE_CONTROLLER) != null) {
-            nodesNeedingRestart = partitioned.get(NodeFlavour.BROKER_AND_NOT_ACTIVE_CONTROLLER).stream()
-                    .filter(entry -> !entry.getValue().controller() || isQuorumHealthyWithoutNode(reconciliation, entry.getKey(), activeControllerId, totalNumOfControllerNodes, quorumState))
+        } else if (partitioned.get(NodeFlavour.BROKER) != null) {
+            nodesNeedingRestart = partitioned.get(NodeFlavour.BROKER).stream()
                     .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
             if (nodesNeedingRestart.isEmpty()) {
-                LOGGER.warnCr(reconciliation, "The combined nodes {} cannot be restarted without impacting the quorum health", partitioned.get(NodeFlavour.BROKER_AND_NOT_ACTIVE_CONTROLLER));
+                LOGGER.warnCr(reconciliation, "The combined nodes {} cannot be restarted without impacting the quorum health", partitioned.get(NodeFlavour.COMBINED_AND_NOT_ACTIVE_CONTROLLER));
```

Comment:
Do we need this line? We probably can remove it since it's not combined node anymore and not doing quorum check.